timesys/high-level-triggers.py

```diff
@@ -1748,6 +1757,232 @@
+
+            'SI-09M1:DI-BPM:CRT7', 'SI-09M2:DI-BPM:CRT7',
+            'SI-09C1:DI-BPM-1:CRT7', 'SI-09C1:DI-BPM-2:CRT7',
+            'SI-09C2:DI-BPM:CRT7', 'SI-09C3:DI-BPM-1:CRT7',
+            'SI-09C3:DI-BPM-2:CRT7', 'SI-09C4:DI-BPM:CRT7',
+
+            'SI-10M1:DI-BPM:CRT7', 'SI-10M2:DI-BPM:CRT7',
+            'SI-10C1:DI-BPM-1:CRT7', 'SI-10C1:DI-BPM-2:CRT7',
+            'SI-10C2:DI-BPM:CRT7', 'SI-10C3:DI-BPM-1:CRT7',
+            'SI-10C3:DI-BPM-2:CRT7', 'SI-10C4:DI-BPM:CRT7',
+
+            'SI-11M1:DI-BPM:CRT7', 'SI-11M2:DI-BPM:CRT7',
+            'SI-11C1:DI-BPM-1:CRT7', 'SI-11C1:DI-BPM-2:CRT7',
+            'SI-11C2:DI-BPM:CRT7', 'SI-11C3:DI-BPM-1:CRT7',
+            'SI-11C3:DI-BPM-2:CRT7', 'SI-11C4:DI-BPM:CRT7',
+
+            'SI-12M1:DI-BPM:CRT7', 'SI-12M2:DI-BPM:CRT7',
+            'SI-12C1:DI-BPM-1:CRT7', 'SI-12C1:DI-BPM-2:CRT7',
+            'SI-12C2:DI-BPM:CRT7', 'SI-12C3:DI-BPM-1:CRT7',
+            'SI-12C3:DI-BPM-2:CRT7', 'SI-12C4:DI-BPM:CRT7',
+
+            'SI-13M1:DI-BPM:CRT7', 'SI-13M2:DI-BPM:CRT7',
+            'SI-13C1:DI-BPM-1:CRT7', 'SI-13C1:DI-BPM-2:CRT7',
+            'SI-13C2:DI-BPM:CRT7', 'SI-13C3:DI-BPM-1:CRT7',
+            'SI-13C3:DI-BPM-2:CRT7', 'SI-13C4:DI-BPM:CRT7',
+
+            'SI-14M1:DI-BPM:CRT7', 'SI-14M2:DI-BPM:CRT7',
+            'SI-14C1:DI-BPM-1:CRT7', 'SI-14C1:DI-BPM-2:CRT7',
+            'SI-14C2:DI-BPM:CRT7', 'SI-14C3:DI-BPM-1:CRT7',
+            'SI-14C3:DI-BPM-2:CRT7', 'SI-14C4:DI-BPM:CRT7',
+
+            'SI-15M1:DI-BPM:CRT7', 'SI-15M2:DI-BPM:CRT7',
+            'SI-15C1:DI-BPM-1:CRT7', 'SI-15C1:DI-BPM-2:CRT7',
+            'SI-15C2:DI-BPM:CRT7', 'SI-15C3:DI-BPM-1:CRT7',
+            'SI-15C3:DI-BPM-2:CRT7', 'SI-15C4:DI-BPM:CRT7',
+
+            'SI-16M1:DI-BPM:CRT7', 'SI-16M2:DI-BPM:CRT7',
+            'SI-16C1:DI-BPM-1:CRT7', 'SI-16C1:DI-BPM-2:CRT7',
+            'SI-16C2:DI-BPM:CRT7', 'SI-16C3:DI-BPM-1:CRT7',
+            'SI-16C3:DI-BPM-2:CRT7', 'SI-16C4:DI-BPM:CRT7',
+
+            'SI-17M1:DI-BPM:CRT7', 'SI-17M2:DI-BPM:CRT7',
+            'SI-17C1:DI-BPM-1:CRT7', 'SI-17C1:DI-BPM-2:CRT7',
+            'SI-17C2:DI-BPM:CRT7', 'SI-17C3:DI-BPM-1:CRT7',
+            'SI-17C3:DI-BPM-2:CRT7', 'SI-17C4:DI-BPM:CRT7',
+
+            'SI-18M1:DI-BPM:CRT7', 'SI-18M2:DI-BPM:CRT7',
+            'SI-18C1:DI-BPM-1:CRT7', 'SI-18C1:DI-BPM-2:CRT7',
+            'SI-18C2:DI-BPM:CRT7', 'SI-18C3:DI-BPM-1:CRT7',
+            'SI-18C3:DI-BPM-2:CRT7', 'SI-18C4:DI-BPM:CRT7',
+
+            'SI-19M1:DI-BPM:CRT7', 'SI-19M2:DI-BPM:CRT7',
+            'SI-19C1:DI-BPM-1:CRT7', 'SI-19C1:DI-BPM-2:CRT7',
+            'SI-19C2:DI-BPM:CRT7', 'SI-19C3:DI-BPM-1:CRT7',
+            'SI-19C3:DI-BPM-2:CRT7', 'SI-19C4:DI-BPM:CRT7',
+
+            'SI-20M1:DI-BPM:CRT7', 'SI-20M2:DI-BPM:CRT7',
+            'SI-20C1:DI-BPM-1:CRT7', 'SI-20C1:DI-BPM-2:CRT7',
+            'SI-20C2:DI-BPM:CRT7', 'SI-20C3:DI-BPM-1:CRT7',
+            'SI-20C3:DI-BPM-2:CRT7', 'SI-20C4:DI-BPM:CRT7',
```

Comment:
Shouldn't the TL BPMs be here as well? AFAIK their sync mechanism depends on the monit clock as well.

Reply:
Is it important that they are here? In this case wouldn't it be important to have something similar for the booster?

Reply:
Indeed, since they are all synced by the same signal (or at least, supposed to be). Though we also operate in direct mode instead of switching, which makes syncing FOFB a bit weird, AFAIU.

For both the TLs and booster, we mainly use the TbT and ADC rates, right?

Reply:
For the TLs we use only ADC rate, but for the booster we also use the FOFB and FAcq rates.

Reply:
Are the ADC and TbT rates synced with the same trigger too?

Reply:
I added TB, TS and BO to the trigger to.